templates/zerver/help/organize-the-streams-sidebar.md

```diff
@@ -0,0 +1,56 @@
+# Organizing the Streams sidebar
+
+On Zulip, it is useful to organize streams in the sidebar in order to easily
+ access streams which you use frequently. This is because streams are initially
+ arranged alphabetically which makes it difficult to quickly access your
+ frequently used streams.
+
+In order to organize streams, you can use 3 methods: pinning, searching and muting.
+
+## Pinning
+Pinning a stream on Zulip pushes the stream to the top of your **Streams**
+ sidebar and sticks/pins it there so that it stays at the top of your **Streams**
+ sidebar.
+
+{!stream-actions.md!}
+
+2. Click on the {!down-chevron.md!}
+
+3. Select the **Pin stream (stream name) to top** option from the actions dropdown.
+
+4. The stream will now stay at the top of your **Streams** sidebar
+
+    !!! tip ""
+        You can unpin a pinned stream by repeating steps 1 and 2 and then selecting
+         **Unpin stream (stream name) from top**
+
+## Searching
+ If don't have a stream pinned, you can search for it instead of scrolling
+ through the alphabetically ordered list.
+
+1. On the left sidebar in the **Streams** section, click on the search
+ (<i class="fa fa-search" aria-hidden="true"></i>) icon in the top right
+ hand corner.
+
+    !!! tip ""
+        You can also use the hotkey "w" to open the search box directly
+
+2. A search box entitled **Search streams** will appear. You can type the
+ name of a stream into this box and the **Streams** section will be filtered
+ accordingly.
+
+## Muting
+
+If you want to keep a stream on your sidebar but don't want to keep receiving
+ notifications from it, you can mute the stream.
+
+{!stream-actions.md!}
+
+2. Click on the {!down-chevron.md!}
+
+3. Select the **Mute the stream (stream name)** option from the actions dropdown.
+
+4. The stream will now be greyed out and you will no longer receive
```

Comment:
grayed instead of greyed for docs consistency. (I was corrected on this as well)